package.json

```diff
@@ -28,5 +29,14 @@
   "devDependencies": {
     "enzyme": "^3.8.0",
     "enzyme-adapter-react-16": "^1.9.1"
+  },
+  "standard": {
+    "parser": "babel-eslint",
```

Comment:
This is a formatting thing with VSCode, I'm guessing. It might be needed for the PR code to play nice with your VSCode formatting, so I'll allow it in this PR, but I'm removing the irrelevant portions of the config:
```
    "ignore": [
      "src/assets/*"
    ],
    "env": [
      "jest"
    ]
```
We can add them back as needed.